tty/tty.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"github.com/sttk-go/sabi"
+	"os"
+)
+
+const (
+	MODE_ERROR = iota - 1
+	MODE_NORMAL
+	MODE_SILENT
+	MODE_HELP
+	MODE_VERSION
+)
+
+type /* error reasons */ (
+	InvalidOption struct{ Option string }
+	StdinIsNotTty struct{}
```

Comment:
`StdinIsNotTty` isn't associated with `FailToGetTtyName` for `TtyDax`.

Redefine `TtyDax` as `Conn` (i.e. `TtyConn`) and make another implementation of `TtyDax` in `tty`'s `main` package.